packages/babel-plugin-material-ui/test/__fixtures__/core/components/output.js

```diff
@@ -0,0 +1,3 @@
+import MuiAppBar from "@material-ui/core/AppBar";
+import Grid from "@material-ui/core/Grid";
+import Modal, { ModalManager } from "@material-ui/core/Modal";
```

Comment:
Oh boy, we export the ModalManager 😨. It should be private, I have added in the v5 BC wishlist.

Reply:
Yeah, I was wondering why it was public but not documented